### ISSUE TITLE
Update app to use node 16 and npm 8

### ISSUE THREAD

**Issue by sarinapadilla** [NCIOCPL/cgov-react-app-playground#56]
### Issue description
Update the `engines` in package.json to the new versions, and update the GH workflow. We will also have to update package-lock.json to use the new npm format.

As part of this ticket, we need to update the shared workflow to use the node 18 upon installation.

> **ESTIMATE** 2

### Additional details / screenshot
- ![Screenshot]()

### Related Tickets
* Dependent on #74 
* Original Story Ticket #9999

**Comment by alerosad** [comment]
upgrading to node 17 or higher seems to break the app, currently trying to resolve.

**Comment by sarinapadilla** [comment]
This is a duplicate of #74 , so closing this ticket.
